nav2_bt_navigator/behavior_trees/README.md

```diff
@@ -0,0 +1,12 @@
+# Instructions on using Groot
+[Groot](https://github.com/BehaviorTree/Groot) is the companion application of [BehaviorTree.CPP](https://github.com/BehaviorTree/BehaviorTree.CPP) to create, edit, and monitor behavior trees.
+
+##### Note: Currently only supports visualization of the behavior tree along with the creation of new nodes (except ControlNodes) using the Groot. Additional features under development.
```

Comment:
good note, but maybe make it more clear. Is this just saying that custom control nodes can't be used? Its a little unclear if there's anything else missing or if that's just it from this note.

Reply:
Custom ControlNodes can't be created using Groot but can be done manually. For now, I think I should change it to:
> Currently only supports visualization of the behavior trees. Additional features under development.

As I haven't provided instructions to create custom nodes using Groot. Waiting for native support for creating custom ControlNodes before working on that.

Reply:
How about:



    Currently fully supports visualization of the behavior trees. It also supports creation of custom nodes except control flow nodes,  which are under development.


I want the boundaries of what it can and cannot do to be crystal clear

Reply:
Oh, I also meant real time monitoring using Groot is also not yet supported. ControlNodes can be added manually for visualization purposes.

> Currently fully supports visualization of the behavior trees. It also supports the creation of custom nodes except control flow nodes. Support for custom control flow nodes and real-time monitoring support is under development.

Is this clear now?

Reply:
Yes, that's good!